test/test_helper.rb

```diff
@@ -16,6 +16,15 @@
 require 'gds_api/test_helpers/content_store'
 require 'gds_api/test_helpers/rummager'
 
+# Note that simplecov only reports test coverage for the /tests, not /features.
+if ENV["USE_SIMPLECOV"]
+  require "simplecov"
+
+  SimpleCov.start do
```

Comment:
I think you can just do `SimpleCov.start 'rails'`, and it'll set up all the necessary filters etc. for a Rails app.